src/Internal/Payment/State/AuthenticationRequiredState.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Class AuthenticationRequiredState
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Payment\State;
+
+/**
+ * The state, which indicates that the payment requires authentication (3DS).
+ */
+class AuthenticationRequiredState extends AbstractPaymentState {
```

Comment:
Based on https://github.com/Automattic/woocommerce-payments/pull/7471#discussion_r1361620696: 

```suggestion
class PendingAuthenticationState extends AbstractPaymentState {
```